misk-api/src/main/kotlin/misk/Action.kt

```diff
@@ -38,10 +37,6 @@ data class Action(
       }
     }
 
-  internal inline fun <reified T : Annotation> parameterAnnotatedOrNull(): KParameter? {
```

Comment:
Is this unused everywhere including internally?

Reply:
There were 2 usages of this function. I replaced them inline.